vue/components/ui/atoms/button-icon/button-icon.vue

```diff
@@ -254,6 +258,18 @@ export const ButtonIcon = {
             };
             if (this.color) base["button-icon-" + this.color] = this.color;
             return base;
+        },
+        loaderStyle() {
+            const scaledSize = (this.size * 10) / 30;
+            const scaledBorderWidth = (this.size * 2) / 30;
+            const marginTop = (this.size * 2) / 30; // Fixes the "ball-clip-rotate" animation not being centered

```

Comment:
No inline comments + comments start with lowercase. Also, if you are to add a comment, it must say why does it fix it.